### PR TITLE
Fix maxp extraction for non-TTF

### LINKED ISSUE #33: seems like something is missing
got this error @jenskutilek 

```console
extractor/formats/opentype.py", line 137, in extractMaxpValues
    "maxFunctionDefs": maxp.maxFunctionDefs,
AttributeError: 'table__m_a_x_p' object has no attribute 'maxFunctionDefs'
```

### PR DESCRIPTION
**Pull request by jenskutilek**
- Fixes #33
- Also adds OTF test data

### CODE REVIEW

File: Lib/extractor/formats/opentype.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def extractFontFromOpenType(
         destination.kerning.update(kerning)
     for function in customFunctions:
         function(source, destination)
-    if doInstructions:
+    if doInstructions and "glyf" in source:
```
Comment (typemytype) [inline Lib/extractor/formats/opentype.py:53]: can this if condition be handled in side `extractInstructions` like you did other if statements? 

see https://github.com/robotools/extractor/blob/master/Lib/extractor/formats/opentype.py#L423

File: tests/roundtrip_otf.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import extractor
```
Comment (typemytype) [inline tests/roundtrip_otf.py:1]: this is not really a test

but maybe it should become one, different PR 
I would consider Plex as examples and have smaller scale examples of otf, ttf, pfa, pfb, woff, woff2 to extract and compare it with provided ufo data.

**Review by typemytype — CHANGES_REQUESTED** [review]
thanks! looks good, just one minor change requeted

File: Lib/extractor/formats/opentype.py (hunk 1 above)
Comment (jenskutilek) [reply Lib/extractor/formats/opentype.py]: Sure! I moved the condition.

File: tests/roundtrip_otf.py (hunk 2 above)
Comment (jenskutilek) [reply tests/roundtrip_otf.py:1]: Yeah, I know it's not a proper test. More a script to make sure everything works. I actually began writing proper tests, but as you say, they should better be in a different PR.

**Review by typemytype — APPROVED** [review]

**Comment by typemytype** [comment]
thanks

**Pull request merged**